manifests/config.pp

```diff
@@ -97,4 +97,27 @@
       refreshonly => true,
     }
   }
+
+  if $pulp::manage_squid {
+    if $::osfamily == 'RedHat' and versioncmp($::operatingsystemrelease, '7') < 0 {
+      $deprecated_opts = true
+    } else {
+      $deprecated_opts = false
+    }
+    class { '::squid3':
+      use_deprecated_opts           => $deprecated_opts,
+      http_port                     => [ '3128 accel' ],
+      acl                           => [ 'Safe_ports port 3128' ],
+      http_access                   => [ 'allow localhost', 'deny to_localhost', 'deny all' ],
+      cache                         => [ 'allow all' ],
+      maximum_object_size           => '5 GB',
+      maximum_object_size_in_memory => '100 MB',
```

Comment:
I might would make this smaller, I'm thinking somewhere closer to 1MB.  I'm okay leaving this as is for now and adjusting later, but I think it needs to be tested for memory consumption at some point.